src/crawlee/http_clients/_httpx.py

```diff
@@ -101,6 +102,7 @@ def __init__(
         ignore_http_error_status_codes: Iterable[int] = (),
         http1: bool = True,
         http2: bool = True,
+        verify: str | bool | SSLContext = True,
```

Comment:
I know that this is the name of the attribute in httpx, but maybe we could rename this to something like `verify_certificates` or `verify_ssl` for better clarity?

Reply:
These names are indeed clearer.

But as a weak argument. The `requests` API, has become such a standard for HTTP clients in Python that any Python developer, will expect exactly `verify`.